### DISCUSSION TITLE
Windows 10 As Network Encoder?

### DISCUSSION THREAD

**Discussion by jchiso** [OpenSageTV/Community#148] [Capture Devices]
A few months ago I started having issues with my Linux-based network encoding server.  I have three Colossus 2 cards in the server. This worked fine for about three years, but I suddenly started to experience repeated capture failures. I had been running OpenDCT for tuner control.

I bit the bullet and converted the server to Windows 10 last week, only to be met by numerous issues.  i think I have overcome most of them, related to external tuning commands and registry edits, but the (Hauppauge) capture software does not initiate captures within Sage.  i can capture from each of the devices with the Hauppauge app, using the most-recent and previous driver.

I did see a post about allowing the capture program to have access to the "Camera", but the issue(s) persist.  The problems seem to be that Windows 10 is preventing the capture device from starting when initiated by a client.  

 I have a Windows 8.1 system that does not seem to have this issue (though it has other intermittent issues).

Are there some elements of Win 10 that make network encoding such a challenge for Sage?

**Comment by wayner9** [comment]
I don't have any advice but I am considering using a Win 10 PC as a network encoder to feed my unRAID Linux system.  The reason for using a network encoder is to be able to use a HDPVR2 as a capture device.  I recently had to switch from old digital cable boxes with component outputs that fed HDPVRs to a new IgniteTV (same as Xfinity) STB that only has an HDMI output.  I have tried a couple of different HDMI->component adapters but the quality is lacking.  The HDPVR2 does not seem to be supported in unRAID Linux as I don't think there are Linux drivers for the HDPVR2 so I can't use it directly.

**Reply by jchiso** [reply]
For what it's worth: there are [Linux drivers and setup instructions](https://www.hauppauge.com/pages/support/support_linux.html#hdpvr2) for the HDPVR2 and Colossus 2 chipset.  I was always able to do HDMI captures with AC3 on my Ubuntu server based on the info on the instructions referenced.  I just lost the ability to have Sage perform the captures ...

**Reply by Zogg44** [reply]
Did you install the Hauppage package as Administrator in Win 10? Are you running the SageTV service in Win 10? If not then you might be running into a permissions problem. It's been a while but I think everything needs to have admin privileges.

Like wayner9, I could never get an HDPVR2 supported on my unRAID system so I also used Windows systems as network encoders. Win 10 can be a bit tricky to get going though due to all the different versions and updates.

**Reply by jchiso** [reply]
I will re-install the Hauppauge package as admin.  That never even occurred to me, but you are correct: Windows updates seem to disrupt previously-functional setups.  I am not running Sage as a service.  I'd forgotten about that option, and don't really recall what it takes to get that going.  I could scour the old forum for notes, allowing for the fact that OS changes may make setup challenging ...

**Reply by Zogg44** [reply]
If you haven't already started the removal/install process, you could try running SageTV with admin privileges and see what happens.

As for the service, I don't remember exactly how to set that up, sorry.

**Reply by wayner9** [reply]
> For what it's worth: there are [Linux drivers and setup instructions](https://www.hauppauge.com/pages/support/support_linux.html#hdpvr2) for the HDPVR2 and Colossus 2 chipset. I was always able to do HDMI captures with AC3 on my Ubuntu server based on the info on the instructions referenced. I just lost the ability to have Sage perform the captures ...

Thanks, but given that I use unRAID I don't know how I deal with this if the drivers aren't part of the core unRAID install or the DVB plugin that seems to give you drivers for other capture devices.

**Comment by CptKirk-1701** [comment]
The SageTV manual has service info on pages 15-17

http://download.sage.tv/SageTV_Manual.pdf

**Reply by jchiso** [reply]
I knew about this from years ago, but I don't know how much of this was still applicable after Windows XP ...

**Comment by jchiso** [comment]
First step of trying to set Sage to 'Run as Administrator' at least has allowed me to start a successful live encode.  I will make that change permanent at startup and see if any other issues arise.  On my Linux-based system the first issue that started was not being able to do scheduled captures, even though live caps worked.  So I'm not prepared do declare victory, just yet ...

**Comment by CptKirk-1701** [comment]
I am running SageTV as a service on my Win10 machine.  Never had any problems.  Pretty straight forward setup.

(NOTE: please scroll to the bottom and 'Add a comment' instead of 'Write a reply'.  It keeps new entries in order, instead of scattered about in the thread.  That's a Github thing and is annoying).
